README.md

```diff
@@ -10,6 +10,8 @@ There is [extensive documentation](https://codeql.github.com/docs/) on getting s
 
 We welcome contributions to our standard library and standard checks. Do you have an idea for a new check, or how to improve an existing query? Then please go ahead and open a pull request! Before you do, though, please take the time to read our [contributing guidelines](CONTRIBUTING.md). You can also consult our [style guides](https://github.com/github/codeql/tree/main/docs) to learn how to format your code for consistency and clarity, how to write query metadata, and how to write query help documentation for your query.
 
+For information on contributing to CodeQL documentation, see "[the contributing guide](https://github.com/github/codeql/blob/main/docs/codeql/CONTRIBUTING.md)" for docs.
```

Comment:
minor: I'm pretty sure this works and it's generally better since if you are viewing this file on a branch or special commit, clicking the link will take you to `CONTRIBUTING.md` on that same commit (instead of main).

```suggestion
For information on contributing to CodeQL documentation, see "[the contributing guide](docs/codeql/CONTRIBUTING.md)" for docs.
```

Reply:
great point, thank you!! Looks like these work 👍